Next 13: build errors

## Package
_Choose from the list:_
- [x] app-content-pages
- [x] app-project


## Describe the bug
A couple of build errors since upgrading to Next 13:
- `eslint-config-next` errors during project builds.
>info  - Linting and checking validity of types ..error - ESLint: Failed to load parser './parser.js' declared in '.eslintrc » eslint-config-next/core-web-vitals » /Users/odonnell/zooniverse/front-end-monorepo/node_modules/eslint-config-next/index.js': Cannot find module 'next/dist/compiled/babel/eslint-parser' Require stack: - /Users/odonnell/zooniverse/front-end-monorepo/node_modules/eslint-config-next/parser.js - /Users/odonnell/zooniverse/front-end-monorepo/node_modules/@eslint/eslintrc/dist/eslintrc.cjs

- [`modularizeImports`](https://nextjs.org/docs/architecture/nextjs-compiler#modularize-imports) is stable, not experimental, in Next 13.4.

## To Reproduce
Build the Next.js apps. Both errors are logged to the console.

--- a/package.json
+++ b/package.json
@@ -53,6 +53,7 @@
       "packages/**"
     ],
     "nohoist": [
+      "**/eslint-config-next",
       "**/@storybook/*",
       "**/storybook",
       "**/storybook-i18n",

--- a/packages/app-content-pages/next.config.js
+++ b/packages/app-content-pages/next.config.js
@@ -44,21 +44,22 @@ const nextConfig = {
   },
 
   experimental: {
-    forceSwcTransforms: true,
-    modularizeImports: {
-      lodash: {
-        transform: 'lodash/{{member}}',
-      },
-      '@zooniverse/react-components': {
-        transform: '@zooniverse/react-components/{{member}}'
-      }
-    },
+    forceSwcTransforms: true
   },
 
   /** localeDetection is a Next.js feature, while the rest of i18n config pertains to next-i18next */
   i18n: {
     localeDetection: false,
     ...i18n
+  },
+
+  modularizeImports: {
+    lodash: {
+      transform: 'lodash/{{member}}',
+    },
+    '@zooniverse/react-components': {
+      transform: '@zooniverse/react-components/{{member}}'
+    }
   },
 
   reactStrictMode: true,

--- a/packages/app-project/src/components/Head/Head.js
+++ b/packages/app-project/src/components/Head/Head.js
@@ -33,7 +33,9 @@ function Head (props) {
       <link rel='apple-touch-icon' href='/touch-icon.png' />
       <link rel='mask-icon' href='/favicon-mask.svg' color='#49B882' />
       <link rel='icon' href='/favicon.ico' />
-      // preload the classifier's subject placeholder
+      {/*
+        preload the classifier's subject placeholder
+      */}
       <link rel='preload' as='image' href='https://static.zooniverse.org/www.zooniverse.org/assets/fe-project-subject-placeholder-800x600.png' fetchpriority='high' />
 
       <meta property='og:url' content={url} />

